testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/common/Hadoop.java

```diff
@@ -82,8 +84,8 @@ public void extendEnvironment(Environment.Builder builder)
             portBinder.exposePort(container, 50075); // DataNode Web UI prior to Hadoop 3
         });
 
-        builder.configureContainer(
-                COORDINATOR,
+        builder.configureContainersIfPresent(
```

Comment:
I'd reuse existing code:

`builder.configureContainers(container -> if (isPrestoContainer(container.getLogicalName()) {`

Reply:
That was my plan, but then diff was much bigger and it was not clear what changed so introduced a new method. Also I think it is ok to wrap such common use cases as dedicated methods.